lib/activity.php

```diff
@@ -194,8 +192,19 @@ public function getGroupParameter($activity) {
 	 * @return array|false
 	 */
 	public function getNavigation() {
-		return false;
+		$l = \OC::$server->getL10N('passman');
+		return array(
+			'top' => array(),
+			'apps' => array(
+				array(
+					'id' => 'passman',
+					'name' => (string) $l->t('Passwords'),
+					'url' => '',//FIXME: $this->URLGenerator->linkToRoute('activity.Activities.showList', array('filter' => 'passman')),
+				),
```

Comment:
Added the passman filter to the navigation on the left side.
You just need to fix the URL